syntax = "proto3";

message Query {
	string train_file = 1;
	string validation_file = 2;
	string query_wf = 3;
	string answer_wf = 4;
}

message Response {
	float qry_ans_similarity = 5;
	float qry_poor_ans_similarity = 6;
}

service Reply {
	rpc semantic_modeling (Query) returns (Response) {}
}